pkg/apis/resources/v1alpha1/types.go

```diff
@@ -30,6 +30,10 @@ const (
 	// true then the controller will not delete the object in case it is removed from the ManagedResource or the
 	// ManagedResource itself is deleted.
 	KeepObject = "resources.gardener.cloud/keep-object"
+	// Fallback is an annotation that tells the resource manager to ignore the resource if it is
+	// maintained by somebody else. This is identified by looking at the at the description annotation implicitly
+	// added by the resource manager
```

Comment:
Doesn't this break if another GRM manages the resource (e.g., in a shooted seed)?

Reply:
Therefore we should check the owner, also (see comment + use owner of grm /which does not exist)). But basically this is intended for non-resource manager managed entities.

If multiple GRMs try to manage resource they must be coordinated in the way, that only one of them is the leading declaration using a non-fallback resource version